### PR TITLE
Adds a specific RSS feed for each API

### LINKED ISSUE #2865: Add RSS feed for a specific API
# Goal

Generate an individual RSS feed for each public API

# Definition of done

When a user request `/rss/apis/?slug=someslug`, it should receive a specific rss feed for that API

# Related issues

#2310, #2478, #558, #2508, #2521

### PR DESCRIPTION
**Pull request by Vellames**
Closes #2865 

adds a way to generate a specific rss feed for each api

### CODE REVIEW

**Comment by Vellames** [comment]
@mauriciovieira , @matleppa ready for review

File: apinf_packages/apis/collection/server/rss.js [hunk 1]
```diff
@@ -42,8 +42,12 @@ RssFeed.publish('apis', function () {
           before refreshing from the source*/
   feed.setValue('ttl', 1);
 
-  // Look at each entry of Apis shcema and find the latest apis
-  Apis.find({}, { sort: { created_at: -1 } }).forEach((api) => {
+  // Look at each entry of public Apis shcema and find the latest apis
+  const filter = { isPublic: true };
```
Comment (matleppa) [inline apinf_packages/apis/collection/server/rss.js]: Correct the old spelling error shcema --> schema

File: apinf_packages/apis/collection/server/rss.js [hunk 2]
```diff
@@ -42,8 +42,12 @@ RssFeed.publish('apis', function () {
...
+  if (query.slug !== undefined) {
```
Comment (matleppa) [inline apinf_packages/apis/collection/server/rss.js]: Add a comment about using slug as a filter parameter

**Comment by matleppa** [comment]
@mauricio @Vellames Is there a way to verify the functionality of the correction?

**Comment by Vellames** [comment]
@matleppa i did the changes, can you review please?

**Pull request merged**